README.md

```diff
@@ -81,6 +81,10 @@ Set as the `proxy_cache_path` directive in the `nginx.conf` file. By default, th
 
 (For Debian/Ubuntu only) Allows you to set a different repository for the installation of Nginx. As an example, if you are running Debian's wheezy release, and want to get a newer version of Nginx, you can install the `wheezy-backports` repository and set that value here, and Ansible will use that as the `-t` option while installing Nginx.
 
+    nginx_el_install_mainline: yes
```

Comment:
Can you change this variable (just to make it more intuitive) to `nginx_install_redhat_branch` with values of `mainline` or `stable` (and then update the logic in the template file accordingly)? I'd also like the default to be `stable`, since I prefer stability over the latest in this role.